python/private/config_settings.bzl

```diff
@@ -135,8 +166,45 @@ _python_version_major_minor_flag = rule(
     implementation = _python_version_major_minor_flag_impl,
     build_setting = config.string(flag = False),
     attrs = {
-        "_python_version_flag": attr.label(
-            default = _PYTHON_VERSION_FLAG,
-        ),
+        "python_version_flag": attr.label(mandatory = True),
```

Comment:
I don't see why this was made a public attribute?

Reply:
I thought that passing the actual names in the macro itself would be cleaner. This rule is private anyway, so that does not change anything in practise? I will revert the change.

Reply:
Reverting SGTM.